worlds/hexcellsinfinite/docs/setup_en.md

```diff
@@ -0,0 +1,27 @@
+# Hexcells Infinite
+
+## Required Software
+
+- [Hexcells Infinite](https://store.steampowered.com/app/304410/Hexcells_Infinite/)
+- [Hexcells Infinite Randomizer Plugin](https://github.com/Heaxeus/HexcellsInfiniteAP/releases/tag/Release)
+- [BepInEx win_x86 v5](https://github.com/BepInEx/BepInEx/releases)
+- The built-in Archipelago client, which can be installed [here](https://github.com/ArchipelagoMW/Archipelago/releases)
+
+## Connect to the MultiServer
+
+### IMPORTANT!
+### Move/Delete existing slotAP.save files in the Hexcells Infinite/saves directory before connecting to a new multiworld!
```

Comment:
Not about this doc really, but ideally you could identity and load saves based on the seed name so such a thing isn't required (unless you create another room with the same seed, although I don't think many implementations really try to handle that).